suites/reef/rgw/tier-1_rgw_upgrade_ssl_71GA_to_71x_latest.yaml

```diff
@@ -181,15 +181,15 @@ tests:
         commands:
           - "ceph versions"
 
-  # - test:
-  #     name: Test rgw through CURL
-  #     desc: Test rgw through CURL
-  #     polarion-id: CEPH-83575572
-  #     module: sanity_rgw.py
-  #     config:
-  #       script-name: ../curl/test_rgw_using_curl.py
-  #       config-file-name: ../../curl/configs/test_rgw_using_curl.yaml
-  #       timeout: 500
+  - test:
+      name: Test rgw through CURL
+      desc: Test rgw through CURL
+      polarion-id: CEPH-83575572
+      module: sanity_rgw.py
+      config:
+        script-name: ../curl/test_rgw_using_curl.py
+        config-file-name: ../../curl/configs/test_rgw_using_curl.yaml
+        timeout: 500
```

Comment:
nit: AFAIK, `sanity_rgw.py` does not honor the timeout. Please be mindful of the configuration.

Reply:
For now I have removed timeout from testcase level,
we do have a plan to to set timeout from sanity_rgw.py file, so testcase will take timeout from module itself.
note: this particular testcase will not take time more than 1 hr